SPECS/pytorch/pytorch.spec

```diff
@@ -26,6 +24,7 @@ BuildRequires:  python3-six
 
```

Comment:
can you please bump the release number and add a changelog entry

Reply:
Changed 👍 Thank you Andrew